model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -1025,7 +1026,7 @@ class WorkspaceJsonSupport extends JsonSupport {
 
   implicit val EntityTypeMetadataFormat: RootJsonFormat[EntityTypeMetadata] = jsonFormat3(EntityTypeMetadata)
 
-  implicit val EntityQueryFormat: RootJsonFormat[EntityQuery] = jsonFormat7(EntityQuery)
+  implicit val EntityQueryFormat: RootJsonFormat[EntityQuery] = jsonFormat8(EntityQuery)
```

Comment:
Mini q -- why this change? Does it have to do something with the # of keys in the json?

Reply:
yes, exactly. since we added `entityNameFilter` as a brand new field to `EntityQuery`, we have to change this JsonFormat to be an 8-tuple instead of a 7-tuple. It's a quirk of the `spray-json` library.

Reply:
ah got it -- thanks!

Reply:
Yeah! It's changed due to the additional argument/param to entity query